Dockerfile

```diff
@@ -19,6 +19,9 @@ COPY . /app
 RUN pip wheel --no-cache-dir --wheel-dir=wheels -r requirements.txt
 RUN pip install --no-cache-dir --find-links=wheels -r requirements.txt
 
+# Trigger the Prisma CLI to be installed
+RUN prisma -v
```

Comment:
Hmm, we could make a second Docker container, though not sure if it's worth it.

I don't mind carrying this patch out-of-tree, it's pretty easy for us since there's rarely any merge conflicts. Up to you!

Reply:
could make it a cli arg, similar to how tgi does it - https://huggingface.co/docs/text-generation-inference/basic_tutorials/launcher

Reply:
That would still have the slow startup time though I think, since we're spinning a fresh container all the time. =)

Reply:
fair enough